fetchmail_attach_from_folder/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,2 @@
+- Holger Brunn <hbrunn@therp.nl>
+- Ronald Portier <ronald@therp.nl>
```

Comment:
This module was started by @gurneyalex in 7.0, I think you should continue to respect the authorship.

Reply:
The module was actually started by my former collegue @hbrunn , but as @gurneyalex  contributed some improvements I have no problem with adding him as a contributor.